### ISSUE TITLE
Chapter 10 Resnt18RNN model accuracy is 0.

### ISSUE THREAD

**Issue by say2sarwar** [PacktPublishing/PyTorch-Computer-Vision-Cookbook#18]
Amazing working thank you for sharing the code.
the accuracy of the model is 0 or 0.05 on validation set.
what's the issues. i just copy your code. 

Thank you for your time, looking forward for reply.

**Comment by mave5** [comment]
@say2sarwar  make sure that the sanity_check flag is set to False and you train the model for enough epochs. Check the loss metric for conversion of your model.

**Comment by nithikpackt** [comment]
Thank you @mravendi for your solution.
